kubetest2-gke/deployer/commandutils.go

```diff
@@ -140,7 +140,7 @@ func runWithOutputAndReturn(cmd exec.Cmd) (string, error) {
 
 	exec.SetOutput(cmd, io.MultiWriter(os.Stdout, &buf), io.MultiWriter(os.Stderr, &buf))
 	if err := cmd.Run(); err != nil {
-		return "", err
```

Comment:
FWIW this is possible to unit test because of our exec shim, I don't want to block fixing this feature, but it might be worth adding a quick set of tests